api4/team.go

```diff
@@ -235,6 +236,60 @@ func getTeamMembersByIds(c *Context, w http.ResponseWriter, r *http.Request) {
 	w.Write([]byte(model.TeamMembersToJson(members)))
 }
 
+func addTeamMember(c *Context, w http.ResponseWriter, r *http.Request) {
+	c.RequireTeamId()
+	if c.Err != nil {
+		return
+	}
+
+	var tm *model.TeamMember
+	var err *model.AppError
+	var team *model.Team
+	if team, err = app.GetTeam(c.Params.TeamId); err != nil {
+		c.Err = err
+		return
+	}
+
+	props := model.MapFromJson(r.Body)
+	userId := props["user_id"]
+	hash := props["hash"]
+	data := props["data"]
+	inviteId := props["invite_id"]
```

Comment:
at ```client4.go```, what will be the element(s) of ```requestBody```? ```teamId``` only, ```userId``` only or ```teamId``` and ```userId```?
```
func (c *Client4) AddTeamMember(teamId, userId, hash, dataToHash, inviteId string) (*TeamMember, *Response) {
	requestBody := map[string]string{
	    "team_id": teamId,
	    "user_id": userId,
	}
	...
}
```
However if including ```userId```, ```userId``` is optional here unless the client will include in the request their own ```userId``` during by hash or by invite id.